src/Button/Button.js

```diff
@@ -26,6 +26,8 @@ type Props = {
   circular?: boolean,
   /** Disables the Button */
   disabled?: boolean,
+  /** Element to be used as the root node - e.g. 'a' can be used to create a link that is styled like a button */
```

Comment:
Use `a` instead of 'a'?

Capitalize "Button".

Reply:
cool